gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/Concat.feature

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassMap @StepConcat
+Feature: Step - concat()
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectXnull_a_b_nullX_concat
+    Given the modern graph
+    And the traversal of
+      """
+      g.inject(null, "a", "b", null).concat();
```

Comment:
it is better to use data from `Modern` graph, not `inject` because it not supported by some providers.

Reply:
Yes, I will need to add additional tests that uses the modern graph. I see other tests like mean/sum has these inject tests so I think I'll leave these ones for now.